### PR TITLE
add tokenlist.json

### PR DESCRIPTION
**Pull request by CCWorld6**
## **Please provide the following information for your token.**

Please include change to the `tokenlist.json` file in the PR.
DON'T modify any other token on the list.

At minimum each entry should have

- Token Address:
- Token Name:
- Token Symbol:
- Token Decimal:
- Logo URI: 
- Link to the official homepage of token:
- MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
- Existing Markets (where to trade):

### CODE REVIEW

**Review by axelcabee — APPROVED** [review]

**Review by 01SMD — APPROVED** [review]

**Review by 01SMD — APPROVED** [review]

**Comment by mail731212** [comment]
> So what do I do next from here?
> 
> 
>

**Comment by Etverbumcarofactum** [comment]
> ## **Please provide the following information for your token.**
> 
> Please include change to the `tokenlist.json` file in the PR.
> DON'T modify any other token on the list.
> 
> At minimum each entry should have
> 
> - Token Address:
> - Token Name:
> - Token Symbol:
> - Token Decimal:
> - Logo URI: 
> - Link to the official homepage of token:
> - MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
> - Existing Markets (where to trade): 
>

**Comment by sharonleerichardson88** [comment]
Token Address:TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t 
Token Name:TRC20
Token Symbol:USDT
Token Decimal:10
Logo URI: tronscan/tron-tvc-list
Link to the official homepage of token:
MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
Existing Markets (where to trade):

**Comment by sharonleerichardson88** [comment]
Token Address: TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t
Token Name: TRC20
Token Symbol:  USDT
Token Decimal: 10
Logo URI: https://www.google.com/url?q=https://elderu.com/g79Yzvn2%235705dwdbydy02eot9r2zlcyxcn13i1hul86pui&sa=D&source=editors&ust=1701597324156334&usg=AOvVaw2eAmGe6gZjsGX3eMQD6wJ2
Link to the official homepage of token:
MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
Existing Markets (where to trade):

**Comment by nasir677** [comment]
plz add submitt my token

**Review by tino135 — APPROVED** [review]
Mio

**Comment by Vahidki** [comment]
https://github.com/tronscan/tron-tvc-list/pull/61#issuecomment-1869505780https://github.com/tronscan/tron-tvc-list/pull/61#issuecomment-1837417278https://github.com/tronscan/tron-tvc-list/pull/61#issuecomment-1837415939

**Review by TDXCHAIN — APPROVED** [review]
{
      "address": "TKz6qX1NuppMxVwkX7kP3Z7m8CTxibsLGn",
      "symbol": "TDX",
      "name": "TDXchain",
      "decimals": 6,
      "logoURI": "https://photos.app.goo.gl/6VF9iATDzwTaFAiD7",
      "homepage": "https://TDXCHAIN.org/",
      "MarketCapLink": "https://tronscan.com/currencies/TDXCHAIN",
      "existingMarkets": [
          {
              "source": "TRONSCAN",
              "pairs": [
                  "TDX/USDT",
                  "TDX/TRX",
                  "TDX/SUN",
                  "TDX/USDC"
              ]
          },
          {
              "source": "TDXCHAIN",
              "pairs": [
                  "TDX/USDT"
              ]
          },
          {
              "source": "TDXCHAIN",
              "pairs": [
                  "TDX/USDT"
              ]
          }
    ]
}

**Comment by TDXCHAIN** [comment]
> ## **Please provide the following information for your token.**
> 
> Please include change to the `tokenlist.json` file in the PR.
> DON'T modify any other token on the list.
> 
> At minimum each entry should have
> 
> - Token Address:
> - Token Name:
> - Token Symbol:
> - Token Decimal:
> - Logo URI: 
> - Link to the official homepage of token:
> - MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
> - Existing Markets (where to trade): 
> 

{
      "address": "TKz6qX1NuppMxVwkX7kP3Z7m8CTxibsLGn",
      "symbol": "TDX",
      "name": "TDXchain",
      "decimals": 6,
      "logoURI": "https://photos.app.goo.gl/6VF9iATDzwTaFAiD7",
      "homepage": "https://TDXCHAIN.org/",
      "MarketCapLink": "https://tronscan.com/currencies/TDXCHAIN",
      "existingMarkets": [
          {
              "source": "TRONSCAN",
              "pairs": [
                  "TDX/USDT",
                  "TDX/TRX",
                  "TDX/SUN",
                  "TDX/USDC"
              ]
          },
          {
              "source": "TDXCHAIN",
              "pairs": [
                  "TDX/USDT"
              ]
          },
          {
              "source": "TDXCHAIN",
              "pairs": [
                  "TDX/USDT"
              ]
          }
    ]
}

**Review by Heberflaiz — APPROVED** [review]

**Comment by TDXCHAIN** [comment]
> ## **Please provide the following information for your token.**
> 
> Please include change to the `tokenlist.json` file in the PR.
> DON'T modify any other token on the list.
> 
> At minimum each entry should have
> 
> - Token Address:
> - Token Name:
> - Token Symbol:
> - Token Decimal:
> - Logo URI: 
> - Link to the official homepage of token:
> - MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
> - Existing Markets (where to trade): 
> 

@TDXCHAIN

**Comment by kiko1842** [comment]
> ## **Please provide the following information for your token.**
> 
> Please include change to the `tokenlist.json` file in the PR.
> DON'T modify any other token on the list.
> 
> At minimum each entry should have
> 
> - Token Address:
> - Token Name:
> - Token Symbol:
> - Token Decimal:
> - Logo URI: 
> - Link to the official homepage of token:
> - MarketCap Link if available (https://coinmarketcap.com/currencies/#TOKEN or https://www.coingecko.com/en/coins/#TOKEN):
> - Existing Markets (where to trade): 
> 

TCzUYnFSwtH2bJkynGB46tWxWjdTQqL1SG
APENFT
Trc721
6
Tron

**Review by kiko1842 — APPROVED** [review]

**Review by kiko1842 — APPROVED** [review]

**Review by kiko1842 — APPROVED** [review]

**Review by kiko1842 — APPROVED** [review]

**Review by kiko1842 — APPROVED** [review]

**Review by kiko1842 — APPROVED** [review]

**Review by kiko1842 — APPROVED** [review]

**Comment by grap4082** [comment]
Please add my Bitcoin Wallet account with merging on expedited request upon approval with transferring my cryptocurrency into accounts thank you lnbc1pn2s8rydqdgdshx6pqg9c8qpp5lr2whteqsgx7u85f6x8cnkxxx9np32ce46ayjc9y26j8pnxhev8ssp5ed8t8ssfqq9yd78ste0tq8ej3cqg2drv53gm8775ln4wfqnkgg3s9qrsgqcqpcxqy8ayqrzjqv06k0m23t593pngl0jt7n9wznp64fqngvctz7vts8nq4tukvtljqzyjvcqq32cqquqqqqqqqqqqqqqq9grzjqtsjy9p55gdceevp36fvdmrkxqvzfhy8ak2tgc5zgtjtra9xlaz97zz6suqq2wgqq5qqqqqqqqqqqqqq9g7vpahyjm5e34jz4g76zkyhempfre6ewsdtm53yafky2f0jgtex4xcta7xwxc5hpseu77ml86su5wje0mv0cp78728q5xh5f0lu58p7spltljje

**Review by grap4082 — APPROVED** [review]

**Review by grap4082 — APPROVED** [review]
Please redepository into this transmission Bitcoin wallet on account thank you it's already linked with my account here and wanted to deposit merge my account in with redepository for deposit thank you bc1q8xpnsekdslsgjgudlgl0s5g0q03sp3gaez9q3r

**Review by grap4082 — APPROVED** [review]
And take 3% or 4% off of the price of the market of Bitcoin for you guys's Credit Central market value on each and every cryptocurrency depository market thank you please deposit here into my account thank you bc1q8xpnsekdslsgjgudlgl0s5g0q03sp3gaez9q3r

**Review by Heberflaiz — APPROVED** [review]

**Review by Heberflaiz — APPROVED** [review]

**Comment by grap4082** [comment]
Thanks a lot.

**Review by grap4082 — APPROVED** [review]

**Review by tawhidnazari57 — COMMENTED** [review]
TXZ7YgZPSt8mpzwAu3b19TvhwP2HLXL9Ty

File: tokenlist.json [hunk 1]
```diff
@@ -1,3 +1,17 @@
-[
-
```
Comment (tawhidnazari57) [inline tokenlist.json:2]: TXZ7YgZPSt8mpzwAu3b19TvhwP2HLXL9Ty

File: tokenlist.json [hunk 2]
```diff
@@ -1,3 +1,17 @@
-[
```
Comment (tawhidnazari57) [inline tokenlist.json:1]: TRC20

File: tokenlist.json [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
-    
-]
```
Comment (tawhidnazari57) [inline tokenlist.json:3]: Deposit address

**Review by anfpara-source — COMMENTED** [review]
__

File: tokenlist.json (hunk 1 above)
Comment (meemijung) [reply tokenlist.json:2]: TRO20

**Review by tawhidnazari57 — COMMENTED** [review]
Binance.org

**Review by MoeGawad10 — COMMENTED** [review]
Address added it

**Review by MoeGawad10 — COMMENTED** [review]
17+ token outside

**Comment by Osamaabdalsabour** [comment]
تم

في الاثنين، ٢٦ أغسطس ٢٠٢٤, ١٠:٥٥ م MoeGawad10 ***@***.***>
كتب:

**Review by tawhidnazari57 — COMMENTED** [review]
0xd5152b4fd45dbf4b802c3e57a2038ad82ac2d688

**Review by Chator1 — COMMENTED** [review]
I would like to merge my wallet and deposit
0xFD689e5f2d8d9Aec0aD328225Ae62FdBDdb30328
TZ57Hd4kz8cGJRA5FAYd5iK8PykBkQ761P
0x345d939cb9cac48ede2e0f2e59a9e98e461ff6c7

File: tokenlist.json [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
\ No newline at end of file
+{
```
[the remark this answers is not in the document]
Comment (Osamaabdalsabour) [reply tokenlist.json:1]: وما الذي يمنعك عن دمجها ماهى المشاكل التى تقابلها رجاء اشرح مشكلتك بالتفصيل

[the remark this answers is not in the document]
Comment (Chator1) [reply tokenlist.json:1]: I don't know how

**Review by Elenore23 — COMMENTED** [review]
ja thx guyss for thinking little bit with me, i was active op TRON end 2018 begin 2019, then was that samtings new i interesting so i sign me in 6 jaars ago and today i realize that my little gesture for support for TRON celebrty give to me sametings to doe not just werk and time goes sooo slouly next to me tron is the future with all crypto universe and ai am littl paart off all that.  THX FOR YOU AND EVERYBODY HOW REALIZE .....FUTURE IS JUIST BEHIND THE CORNER...I Love you all and my big respect---

**Comment by Elenore23** [comment]
Beautifullll

**Comment by Chator1** [comment]
Tron web address (TZ57Hd4kz8cGJRA5FAYd5iK8PykBkQ761P)

**Review by 3840986 — COMMENTED** [review]
批准

**Comment by Elenore23** [comment]
Please don't remove tokens from the list..

**Review by Lmillan123 — APPROVED** [review]

**Review by Lmillan123 — APPROVED** [review]

**Comment by LukeLarge** [comment]
Aproved

**Comment by 1341314** [comment]
> و 3% یا 4% از قیمت بازار بیت کوین را برای ارزش بازار Credit Central خود در هر بازار سپرده گذاری ارزهای دیجیتال دریافت کنید. متشکرم لطفاً لطفاً اینجا را به حساب من واریز کنید با تشکر bc1q8xpnsekdslsgjgudlgl0s5g0q03sp3gaez9q3r

اوکی

**Comment by 1341314** [comment]
ممنون

**Comment by Charlesnichahaza** [comment]
Hello

**Review by 1Charmjoe — APPROVED** [review]

**Comment by Kimphung97** [comment]
7d03e46c10ee1c8724c9d2615d19ad9442a3b602

File: tokenlist.json (hunk 3 above)
Comment (1341314) [reply tokenlist.json:3]: TM3sTVyahiGWYktKg8G6miHpTzRurKDt7b

**Review by 1341314 — COMMENTED** [review]
TM3sTVyahiGWYktKg8G6miHpTzRurKDt7b

**Review by Fishy2k21 — COMMENTED** [review]
24b883458e31f02b1d93f7449ee6fe326cac8b95

**Comment by anfpara-source** [comment]
sTOP

**Comment by b0900062030** [comment]
> TXZ7YgZPSt8mpzwAu3b19TvhwP2HLXL9Ty

File: tokenlist.json (hunk 1 above)
Comment (b0900062030) [reply tokenlist.json:2]: 安

File: tokenlist.json (hunk 2 above)
Comment (b0900062030) [reply tokenlist.json:1]: 安

**Review by b0900062030 — APPROVED** [review]

**Review by b0900062030 — APPROVED** [review]

**Review by b0900062030 — APPROVED** [review]

**Comment by Hgaras** [comment]
TQ9cP2TTKM4aPTztg8HACNNzdzhANPG5E4

**Comment by galaxycores** [comment]
Mostafa#6

**Review by Armss9936 — APPROVED** [review]

**Review by amiahudson7-cmyk — APPROVED** [review]

**Review by hasoneh7 — COMMENTED** [review]
Mohammad hasoneh

**Review by hasoneh7 — COMMENTED** [review]
Hanoda.crazy1994@gmail.com

File: tokenlist.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (hasoneh7) [reply tokenlist.json:1]: لا استطيع

**Comment by jaroslavsonce-prog** [comment]
Добрий день я не розумію що маю зробити

**Review by hasoneh7 — COMMENTED** [review]
Hasoneh

File: tokenlist.json [hunk 5]
```diff
@@ -1,3 +1,17 @@
...
+    "address": "TW63guytLiXrGU9KrVgduoHRXaubaUkn3T",
```
Comment (hasoneh7) [inline tokenlist.json:2]: Hasoneh

**Review by AlirezaDarbandi — APPROVED** [review]

File: tokenlist.json (hunk 2 above)
Comment (newalemon-pixel) [reply tokenlist.json:1]: Why

**Pull request merged**